### ISSUE TITLE
Suggestion: Player teamwork score

### ISSUE THREAD

**Issue by Spshulem** [RaiderIO/raiderio-addon#137]
Mythics are a blast, but time and time again do I run with a DPS/Tank/Healer that is just a bad sport. They'll go AFK, leave the group if ANYONE does anything wrong, pull too much, agro (as a non-tank), and just generally make running the instance harder and unpleasant to run.

All we have to go off of is their Raider.io and we can't really tell if they are actually good to play with or know the runs. So many people get run through instances to gear up, and so the .IO doesn't really help too much.

Additionally, there's no way to report people for being toxic in a group. I've had a group of people all 470, join a group, blitz the first boss for a rare drop, then all drop out (for a Heroic/M0).

It doesn't have to be negative, I would love a way, similar to Overwatch, to rate people as good team players, good at communicating, and other things that we can reward people for. Additionally, we can also downvote people for being a bad team player, toxic, leaving, etc.

Maybe this can be built into Raider.io ... but when I got my friend to join WoW, I can't tell you how many times he's wanted to quit because we spent 45m+ in a mythic only for the tank/healer or DPS to leave because of something silly. Just this morning, I ran 3 mythics, with only 1 that finished, one of which disbanded after the first boss because he had just joined WoW again and had no idea the run (I asked before starting if anyone had any questions).

As u/Dalarrus points out: "perhaps just adding a note to show what % of M+ dungeons the person has done they quit out of instead of completing could help indicate serial leavers."

Would love your thoughts on this... because toxic players really ruin a night playing.

**Comment by Mailliw102** [comment]
Not part of the raiderio team but from my limited experience with addons:
A thumbs up/thumbs down system is not feasible from an addon. Addons can't contact web servers such as raider.io that would process and store this information, and there is no way in game to propagate this player-generated data to everybody that uses the raider.io addon. For something like this to work, it would have to be officially supported in game by Blizzard. As for leaving dungeons early, as far as i know this information is not trackable by raider.io or other websites, it's just not recorded and made available by the game on the official armory/leaderboards.

**Comment by WFrancois** [comment]
Heyas ! 

This kind of feature is indeed really hard to create as as Mailliw102 said, Addons can't contact web server. As such, it is really difficult to create a feature like this, such as no one can tamper with the data being sent to RaiderIO's server.

For % of M+ quit (as Mailliw102 said as well), it's not trackable via World of Warcraft's official API, so can't be implemented :/

**Comment by Spshulem** [comment]
@Mailliw102 @WFrancois thanks for the time. That's a bummer. Hopefully blizzard will do something to help.
